server/statsPageRedis.js

```diff
@@ -75,15 +75,37 @@ module.exports = function(app, env) {
     try {
       const stats = await getStats()
 
+      const tableOfContents = Object.keys(stats)
+        .map(
+          (subject, i) => `
+          <b>${subject}:</b>
+          ${Object.keys(stats[subject])
+    .map((period, j) =>
+      j % 3 === 0
```

Comment:
I would not limit the table of contents only to the first period - it's imho fine to have them all there (especially given how many daily records are there before the monthly ones) and save this condition

Reply:
I decided to do this for two reasons, if the table contained every period it seemed too long. The second reason was that once you see the total number, which is just one line, you always see the whole monthly stats(just couple of lines) and at least the beginning of daily stats. I can't imagine someone going to the table of contents and looking for daily stats if the only thing he had to do is to scroll just a little bit or not at all. What do you think? I have no problem removing the condition if you think it makes more sense.

Reply:
I see - still, I found the table quite straightforward to navigate even if there were all the intervals. There are already like 20 months in mainnet adalite, which takes a significant portion of the viewport and may be annoying to scroll over - and the daily stats are arguably more interesting than monthly or total as they provide the most direct feedback to any changes we did or potential issues we have

A possible tradeoff could be putting the daily, monthly and total link on a single line so they don't take up so much vertical space

Reply:
Agreed, good idea, putting the total, monthly and daily links in on the same line seems like the best solution but it look terrible in mobile design, so I added all the values to the table.